pattern-library/sass/utilities/_variables.scss

```diff
@@ -301,6 +310,7 @@ $spacing-vertical:          (
     small:                  ($baseline),
     x-small:                ($baseline/2),
     xx-small:               ($baseline/4),
+    xxx-small:              ($baseline/10),
```

Comment:
Yeah, maybe we should revisit these shortly to see if we can clean em up (assuming you think they're getting a bit offroad).